fluent-langneg/src/locale.ts

```diff
@@ -1,30 +1,32 @@
-/* eslint no-magic-numbers: 0 */
+function convertMasks(locale: string): string {
+  let result;
+  if (locale[0] === "*") {
+    result = "und" + locale.substr(1);
+  } else {
+    result = locale;
+  };
+  return result.replace(/\-\*/g, "");
+}
 
-const languageCodeRe = "([a-z]{2,3}|\\*)";
-const scriptCodeRe = "(?:-([a-z]{4}|\\*))";
-const regionCodeRe = "(?:-([a-z]{2}|\\*))";
-const variantCodeRe = "(?:-(([0-9][a-z0-9]{3}|[a-z0-9]{5,8})|\\*))";
+function getVisibleLangTagLength(language: any, script: any, region: any) {
```

Comment:
These should really be typed as `string | undefined`; there should be no need for `any` here.